src/main/java/org/gridsuite/shortcircuit/server/entities/FaultResultEntity.java

```diff
@@ -62,6 +62,12 @@ public class FaultResultEntity {
     @Column
     private double ipMin;
 
+    @Column
+    private Double deltaCurrentIpMin;
```

Comment:
Why did you opt for a reference object instead of a primitive type here, especially since these fields are never null and in the mapping you initialize them with Double.NaN?

Reply:
> Why did you opt for a reference object instead of a primitive type here, especially since these fields are never null and in the mapping you initialize them with Double.NaN?

Because if we have old result at the DB (the two values will be null) so when you try to fetch list of FaultResultEntity it's gonna crash( you can assign null values to double). Thats why I keeped reference object.

Reply:
I am not sure that the old values could be null, and moreover, it relies on primitive types